### PR TITLE
Exclude Steam directory itself from backup

### PR DESCRIPTION
**Pull request by nanonyme**

### CODE REVIEW

**Comment by nanonyme** [comment]
@gasinvein created this

File: steam_wrapper/steam_wrapper.py [hunk 1]
```diff
@@ -154,7 +154,8 @@ def migrate_data():
     xdg_data_home = os.path.join(STEAM_ROOT, target)
     if not os.path.islink(source):
         if os.path.isdir(target):
-            copytree(target, backup)
+            copytree(target, backup,
+                     ignore=[os.path.join(target, "Steam")])
```
Comment (gasinvein) [inline steam_wrapper/steam_wrapper.py:158]: `target` here is relative path. Will `fnmatch()` handle it correctly?

Comment (nanonyme) [reply steam_wrapper/steam_wrapper.py:158]: It should. fnmatch won't care even if the file doesn't exist.

Comment (gasinvein) [reply steam_wrapper/steam_wrapper.py:158]: Looks fine to me, then.

**Pull request merged**